client/components/dashboard/dashboard-service.js

```diff
@@ -448,7 +451,7 @@ DashboardService.prototype.selectWidget = function(
 
 /**
  * Scrolls the specified content element (typically a widget or container) into view.
- * @param {!angular.Element} targetElement The element to scroll into view.
+ * @param {!angular.JQLite} targetElement The element to scroll into view.
```

Comment:
I think it's ok since this is basically duck typing. As long as the functionality we're using is part of JQLite and works the same way in full jQuery, this should work. I was following existing examples in client/externs/angular-1.4.js .